### ISSUE TITLE
Documentation points to a 404 page

### ISSUE THREAD

**Issue by DonRichards** [DSpace/DSpace#8788]
**Describe the bug**
This page appears to be out of date https://hub.docker.com/r/dspace/dspace/dockerfile

This refers to a link that results in a 404
<img width="906" alt="Screenshot 2023-04-18 at 9 22 28 AM" src="https://user-images.githubusercontent.com/2738244/232790941-2c5047fa-e87b-4989-a774-0165801b34d9.png">

And it looks like it references 6.x

**To Reproduce**
1. Go to docker hub page listed above.

**Expected behavior**
Not a 404

**Related work**
N/A

**Comment by DonRichards** [comment]
I should add, I found this page by Googling "dspace dockerfile" and it was first link.

**Comment by tdonohue** [comment]
Strangely enough, this Dockerfile shown on DockerHub is non-editable 😞 

I've looked into this, and it appears this is a common complaint with DockerHub. It won't let you select or edit the Dockerfile that it "autodetects".  And it doesn't update automatically unless you build your Docker images *via DockerHub* (we don't do that, we build them in GitHub and push the final images to DockerHub). 

So, we cannot fix this it seems until DockerHub lets us.  I'm a full Admin in our DockerHub acct and I have no way of editing this Dockerfile.

The best I can do is hide the Dockerfile link from the public page (which I just did): https://hub.docker.com/r/dspace/dspace/ 
However, even with that change, the old Dockerfile is still accessible via the URL https://hub.docker.com/r/dspace/dspace/dockerfile  🤷 

 Closing as cannot fix.
